src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs

```diff
@@ -444,5 +444,24 @@ public bool IsAudioOnly
                     ThrowIfFailed(this, "Failed to set the audio-only state of the player");
             }
         }
+
+        private SphericalVideo _sphericalVideo;
+
+        /// <summary>
+        /// Gets the 360 video settings.
+        /// </summary>
+        /// <value>A <see cref="_sphericalVideo"/> that specifies the display settings.</value>
+        /// <since_tizen> 5 </since_tizen>
+        public SphericalVideo SphericalVideo
```

Comment:
If you intend to make this SphericalVideo to singleton, How about consider to change ```internal''' of the access modifier of SphericalVideo? And does it needed to be singleton?

Reply:
because display is singleton, spherical video should be singleton too.
i wonder if i use "internal" instead of "public", can 3rd party developers use SphericalVideo?